<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flnetview</class>
 <widget class="QWidget" name="flnetview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>719</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>flnetview</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QTreeWidget" name="frameDecodeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <property name="showGrid" stdset="0">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Decoded Frame</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPlainTextEdit" name="dataText"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>75</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>Time Stamp</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="labelnWIO">
       <property name="text">
        <string>WIOs: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="textSearch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnSearch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="2">
      <widget class="QLabel" name="labelRecent">
       <property name="text">
        <string>Data/Commnad</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="labelJoined">
       <property name="text">
        <string>Assoc. Req.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Status: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelNever">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>No Activity</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="labelLeft">
       <property name="text">
        <string>Leave Req.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Create Job</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnSearch</sender>
   <signal>clicked()</signal>
   <receiver>flnetview</receiver>
   <slot>onSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSearch()</slot>
 </slots>
</ui>
